packages/ember-glimmer/tests/integration/helpers/element-action-test.js

```diff
@@ -0,0 +1,1340 @@
+    testBoundAction('scissors');
+
+    this.assert.deepEqual(actionOrder, ['rock', 'paper', 'scissors'], 'action name was looked up properly');
+  }
+
+  ['@skip a quoteless string parameter should resolve actionName, including path']() {
+    // TODO: This seems to be covered by the test above. Seems like it should be cut.
+  }
+
+  ['@test a quoteless function parameter should be called, including arguments']() {
+    let submitCalled = false;
+    let incomingArg;
+
+    let arg = 'rough ray';
+
+    let ExampleComponent = Component.extend({
+      submit(actualArg) {
+        incomingArg = actualArg;
+        submitCalled = true;
+      }
+    });
+
+    this.registerComponent('example-component', {
+      ComponentClass: ExampleComponent,
+      template: `<a {{action submit '${arg}'}}>Hi</a>`
+    });
+
+    this.render('{{example-component}}');
+
+    this.runTask(() => {
+      this.$('a').click();
+    });
+
+    this.assert.ok(submitCalled, 'submit function called');
+    this.assert.equal(incomingArg, arg, 'argument passed');
+  }
+
+  ['@test a quoteless parameter that does not resolve to a value asserts']() {
+    let ExampleComponent = Component.extend({
+      actions: {
+        ohNoeNotValid() {}
+      }
+    });
+
+    this.registerComponent('example-component', {
+      ComponentClass: ExampleComponent,
+      template: '<a id="oops-bound-param" {{action ohNoeNotValid}}>Hi</a>'
+    });
+
+    expectAssertion(() => {
+      this.render('{{example-component}}');
+    }, 'You specified a quoteless path to the {{action}} helper ' +
+       'which did not resolve to an action name (a string). ' +
+       'Perhaps you meant to use a quoted actionName? (e.g. {{action \'save\'}}).');
+  }
+
+  ['@glimmer allows multiple actions on a single element']() {
+    let clickActionWasCalled = false;
+    let doubleClickActionWasCalled = false;
+
+    let ExampleComponent = Component.extend({
+      actions: {
+        clicked() {
+          clickActionWasCalled = true;
+        },
+        doubleClicked() {
+          doubleClickActionWasCalled = true;
+        }
+      }
+    });
+
+    this.registerComponent('example-component', {
+      ComponentClass: ExampleComponent,
+      template: strip`
+        <a href="#"
+          {{action "clicked" on="click"}}
+          {{action "doubleClicked" on="doubleClick"}}
+        >click me</a>`
+    });
+
+    this.render('{{example-component}}');
+
+    let actionAttrs = getActionAttributes(this.$('a').get(0));
+
+    this.assert.equal(actionAttrs.length, 2, 'two action attributes were added');
+
+    this.runTask(() => {
+      this.$('a').trigger('click');
+    });
+
+    this.assert.ok(clickActionWasCalled, 'the clicked action was called');
+
+    this.runTask(() => {
+      this.$('a').trigger('dblclick');
+    });
+
+    this.assert.ok(doubleClickActionWasCalled, 'the doubleClicked action was called');
+  }
+
+  ['@htmlbars allows multiple actions on a single element']() {
```

Comment:
Seems like this, and the test above are roughly the same but have a small amount of differences.  It would be nice to either merge them, or somehow mark this `@htmlbars` as something that doesn't need to be "fixed" before enabling glimmer feature flag...